### PR TITLE
Update filter metadata docs

### PR DESCRIPTION
**Pull request by eskibars**
Updates a few things:
- Adds Boolean types, since they're now official
- Adds details on the OOTB metadata filters that you can apply
- Adds cross-referencing of the corpus creation page to the query documentation

Also added to this:
- Booleans
- Exact text matching
- Neural and exact text blending

### CODE REVIEW

File: www/docs/search-apis/exact-boolean-text-matches.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+- Turn off neural retrieval entirely, and instead use exact term matching
+- Incorporate typical keyword modifiers like a `NOT` function, exact phrase
+matching, and wildcard prefixes of terms
+
+# Enabling Exact and Boolean Text Matching
+By default, the exact and Boolean text matching is disabled and only neural
+retrieval is used.  You can enable the feature by specifying a value,
+`lambda`, at query time.  This value can range from `0` to `1` (inclusive).
+
+The default value of `lambda` is `0`, and thus turning off exact and Boolean
```
Comment (amin2718) [inline www/docs/search-apis/exact-boolean-text-matches.md:23]: and thus -> thus

File: www/docs/search-apis/sql/built_in_filterable_metadata.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+id: ootb-filters
+title: Default Filterable Metadata
+---
+
+import {Config} from '../../definitions.md';
+
+A few pieces of metadata that are filterable out of the box, as they're very
```
Comment (amin2718) [inline www/docs/search-apis/sql/built_in_filterable_metadata.md:8]: that are -> are

File: www/docs/search-apis/sql/built_in_filterable_metadata.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+useful in a variety of situations.  This page describes these.
+
+Note tha tyou can set up additional fields to filter on by setting up
```
Comment (amin2718) [inline www/docs/search-apis/sql/built_in_filterable_metadata.md:11]: that you

File: www/docs/search-apis/sql/overview.md [hunk 4]
```diff
@@ -17,6 +17,9 @@ be explicitly specified for every metadata reference in the expression. Valid
 scopes are `doc.` and `part.`, for document and part-level metadata,
 respectively.
 
+To learn more about setting up metadata so that it can be filtered, have a look
+at the [filter attribute][4] section of the
```
Comment (amin2718) [inline www/docs/search-apis/sql/overview.md:21]: Incomplete sentence?

**Review by amin2718 — APPROVED** [review]
Thanks Shane. This is a big improvement.

File: www/docs/search-apis/exact-boolean-text-matches.md (hunk 1 above)
Comment (eskibars) [reply www/docs/search-apis/exact-boolean-text-matches.md:23]: Thanks, fixed

File: www/docs/search-apis/sql/built_in_filterable_metadata.md (hunk 2 above)
Comment (eskibars) [reply www/docs/search-apis/sql/built_in_filterable_metadata.md:8]: Thanks, fixed

File: www/docs/search-apis/sql/built_in_filterable_metadata.md (hunk 3 above)
Comment (eskibars) [reply www/docs/search-apis/sql/built_in_filterable_metadata.md:11]: Thanks, fixed

File: www/docs/search-apis/sql/overview.md (hunk 4 above)
Comment (eskibars) [reply www/docs/search-apis/sql/overview.md:21]: Whoa, weird... I don't know what happened with that one -- must have hit commit in between a copy/paste.  Thanks for flagging.  Fixed

**Comment by eskibars** [comment]
Addressed all comments.  Thanks for the review!

**Pull request merged**